src/DynamoCore/Logging/Analytics.cs

```diff
@@ -19,6 +21,15 @@ internal static void Start(DynamoModel model)
         {
             client = new DynamoAnalyticsClient();
             client.Start(model);
+            model.WorkspaceAdded += OnWorkspaceAdded;
+        }
+
+        static void OnWorkspaceAdded(WorkspaceModel obj)
+        {
+            if (obj is CustomNodeWorkspaceModel)
+                TrackScreenView("CustomWorkspace");
+            else
+                TrackScreenView("Workspace");
```

Comment:
A valid point. Are you proposing that we just need to track custom workspace and skip tracking home workspace. At this point we can't really know if we are in node view or geometry view. Also we are not tracking the Table switching yet.